### ISSUE TITLE
Multiline Parameters: all on one line, or all on new line

### ISSUE THREAD

**Issue by iandundas** [nicklockwood/SwiftFormat#379]
There's a rule in SwiftLint (which doesn't support autocorrect):

[Multiline Parameters](https://github.com/realm/SwiftLint/blob/master/Rules.md#multiline-parameters)

> Functions and methods parameters should be either on the same line, or one per line. 

I don't think it's in SwiftFormat yet, but could be nice to have.

**Comment by nicklockwood** [comment]
@iandundas yeah, I *almost* have support for this, but not quite. The wrapArguments rule is capable of doing it, but currently doesn't have a configuration option to enforce wrapping of every parameter. 

It's something I want to add - I just need to think about how to expose the option.

**Comment by iandundas** [comment]
Awesome! Cool to hear

**Comment by jaredh159** [comment]
@nicklockwood is this still on the roadmap?  would really love to have this feature, which would work like `lineBreakBeforeEachArgument` in `apple-swift-format`.  

Is it something that a new contributor could tackle, if you didn't have time to work on it? I'd be up for it, if you could point me in the right direction.

Thanks for the amazing library, and all your work on it! 🙏

**Comment by jaredh159** [comment]
@nicklockwood or, if, as you say, it's capable of doing it, but just without a config option, is is possible to point out where I could override the setting and build from source until an option is exposed?  Thanks so much!
